### PR TITLE
Add CITATION.cff and update README

### PR DESCRIPTION
**Pull request by murilobalves**
Please check if your names are properly written and if you agree with the order of authors (sorted by number of additions)

### CODE REVIEW

**Review by vellosok75 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Resende"
+  given-names: "Ximenes"
+- family-names: "de Sá"
+  given-names: "Fernando Henrique"
```
Comment (fernandohds564) [inline CITATION.cff:7]: alphabetical order.

**Review by fernandohds564 — CHANGES_REQUESTED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Review by Gabrielrezende-asc — APPROVED** [review]

File: CITATION.cff (hunk 1 above)
Comment (xresende) [reply CITATION.cff:7]: I have very little contribution to this package, in any sense. maybe we should rearrange the order of the authors.

**Review by xresende — CHANGES_REQUESTED** [review]

**Comment by murilobalves** [comment]
what is your suggestion of ordering?

this is the contributors list sorted by additions

image

and this is sorted by commits

image

**Comment by xresende** [comment]
we could sort by commits

**Review by xresende — APPROVED** [review]

**Pull request merged**